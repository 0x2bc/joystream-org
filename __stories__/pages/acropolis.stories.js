import React from 'react';
import { storiesOf } from '@storybook/react';

import AcropolisPage from '../../src/pages/acropolis';

storiesOf('Pages|Acropolis', module).add('default', () => <AcropolisPage />);
