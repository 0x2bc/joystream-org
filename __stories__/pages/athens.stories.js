import React from 'react';
import { storiesOf } from '@storybook/react';

import AthensPage from '../../src/pages/athens';

storiesOf('Pages|Athens', module).add('default', () => <AthensPage />);
