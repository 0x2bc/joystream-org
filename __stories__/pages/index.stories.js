import React from 'react';
import { storiesOf } from '@storybook/react';

import IndexPage from '../../src/pages/index';

storiesOf('Pages|Home', module).add('default', () => <IndexPage />);
