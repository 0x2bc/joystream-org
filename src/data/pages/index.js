import { ReactComponent as ValidatorsImage } from '../../assets/svg/active-validators.svg';
import { ReactComponent as StorageImage } from '../../assets/svg/storage.svg';
import { ReactComponent as MemberImage } from '../../assets/svg/council-member.svg';
import { ReactComponent as ScreenerImage } from '../../assets/svg/screener.svg';
import { ReactComponent as MembershipCuratorImage } from '../../assets/svg/membership-curator.svg';
import { ReactComponent as ContentCuratorImage } from '../../assets/svg/content-curator.svg';
import { ReactComponent as ContentCreatorImage } from '../../assets/svg/content-creator.svg';
import { ReactComponent as BandwidthProviderImage } from '../../assets/svg/bandwidth-provider.svg';
import { ReactComponent as DiscoveryProviderImage } from '../../assets/svg/discovery-provider.svg';
import { ReactComponent as LiveStreamingProviderImage } from '../../assets/svg/live-streaming-provider.svg';
import { ReactComponent as BuilderImage } from '../../assets/svg/builder.svg';
import { ReactComponent as CommunicationModeratorImage } from '../../assets/svg/communication-moderator.svg';

const roles = {
  active: [
    {
      image: ValidatorsImage,
      title: 'Validator',
      to: '/roles#Validator',
      key: 'validatorsCount',
      hasLabel: false,
    },
    {
      image: MemberImage,
      title: 'Council Member',
      to: '/roles#Council-Member',
      key: 'councilMembersCount',
      hasLabel: false,
    },
    {
      image: StorageImage,
      title: 'Storage Provider',
      to: '/roles#Storage-Provider',
      key: 'storageProviders',
      hasLabel: true,
    },
  ],
  future: [
    { image: ScreenerImage, title: 'Membership Screener', to: '/roles' },
    {
      image: MembershipCuratorImage,
      title: 'Membership Curator',
      to: '/roles',
    },
    { image: ContentCuratorImage, title: 'Content Curator', to: '/roles' },
    { image: ContentCreatorImage, title: 'Content Creator', to: '/roles' },
    {
      image: BandwidthProviderImage,
      title: 'Bandwidth Provider',
      to: '/roles',
    },
    {
      image: DiscoveryProviderImage,
      title: 'Discovery Provider',
      to: '/roles',
    },
    {
      image: LiveStreamingProviderImage,
      title: 'Live Streaming Provider',
      to: '/roles',
    },
    { image: BuilderImage, title: 'Builder', to: '/roles' },
    {
      image: CommunicationModeratorImage,
      title: 'Communication Moderator',
      to: '/roles',
    },
  ],
};

export { roles };
