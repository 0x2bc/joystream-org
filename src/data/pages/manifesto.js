export default {
  header: {
    title: 'The Joystream Manifesto',
  },
};
