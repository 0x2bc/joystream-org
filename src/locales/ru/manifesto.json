{
  "manifesto": {
    "siteMetadata": {
      "description": "Прочтите манифест Joystream"
    },
    "hero": {
      "title": "Манифест Joystream",
      "text": "В нашем Манифесте изложены как проблемы, которые, с нашей точки зрения, существуют в индустрии цифровых медиа, так и методы, которые мы предлагаем для их решения."
    },
    "problem": {
      "title": "Проблема",
      "text": {
        "artAsTool": "Мы верим в то, что творчество, будь то литература, музыка, фильмы, игры или перформанс, являются важнейшим инструментом распространения коллективных знаний среди членов сообществ. Эти средства экспрессии определяют общие ценности и, в конечном итоге, участвуют в формировании общей культуры, системы ценностей, и даже политического вектора. Поэтому крайне важную роль играют источники и методы финансирования, а также пути распространения.",
        "mediaAsPrimaryMedium": "На сегодняшний день, цифровые медиа стали основными носителем экспрессий, а медаиплатформы начали играть роль ключевых организаторов этой деятельности.",
        "unaccountableInstitutions": "К сожалению, на сегодняшний день из-за эффектов связанных с функционированием сетей, экономических процессов, регуляций и политических интересов, мы остаемся один на один с несколькими крупными игроками, которые диктуют свои правила. Эти игроки практически полностью подчиняются законам классического рынка и политическим механизмам.",
        "filteredOnTheirTerms": "В итоге, наши сториз, выражения эмоций - любая экспрессия в целом, фильтруется согласно их правилам. Это вытекает в целый калейдоскоп проблем, таких как",
        "inflexible": "негибкая монетизация, цензура высказываний, несправедливое распределение прибыли между создателями контента и аудиторией, низкий уровень инновационности или снижение доверия со стороны сторонних разработчиков."
      }
    },
    "goal": {
      "title": "Цель",
      "text": {
        "arrangement": "Мы считаем, что медиаплатформы обязаны быть подотчетны людям на которых они оказывают влияние, будь то их конечные пользователи, создатели контента, сторонние разработчики или штатные сотрудники.",
        "accountability": "Отсутствие такой формы отчетности и является корнем проблемы."
      }
    },
    "thesis": {
      "title": "Главный тезис",
      "text": {
        "coreThesis": "Наш главный тезис заключается в том, что для достижения цели необходимо решить две основные задачи, и путем к их правильному решению является использование современных распределенных систем и технологий, включая блокчейн, использование смарт контрактов и токенов."
      }
    },
    "wedge": {
      "title": "Препятствия",
      "text": {
        "alternative": "Самым главным вопросом является как создать альтернативную платформу с нуля, учитывая силу внешних факторов, которые сводят эффективность любой новой платформы к нулю, вне зависимости от набора возможностей этой платформы. Исторически сложилось, что процесс такого создания больше напоминает искусство, и успех редко можно повторить.",
        "blockchainTokens": "Мы считаем, что токены, заработанные участниками на раннем этапе работы платформы, показывают свою чрезвычайную эффективность, мобилизуя и мотивируя динамичное сообщество участников из разных стран.",
        "initialHurdle": "В прошлом это помогало зарождающимся сообществам, таким как Bitcoin, преодолеть начальное препятствие сетевого эффекта [1, 2, 3]. И хотя нет никаких гарантий, что этого будет достаточно, в долгосрочной перспективе наша стратегия выглядит для нас многообещающей."
      }
    },
    "accountability": {
      "title": "Масштабирование отчетности",
      "text": {
        "createAndSustain": "Во-вторых: как организовать и поддерживать функционирование такой отчетность, когда масштабы платформы разрастутся.",
        "twoWays": "Согласно А.О. Хиршману [4], существуют два основных метода, используя которые, участники могут привлекать организации любого типа к ответственности: голос и выход."
      }
    },
    "voice": {
      "title": "Голос",
      "text": {
        "improveWithin": "Голос является попыткой улучшить или реформировать структуру организации изнутри. В этом случае эффективность метода зависит как от возможностей обмениваться достоверной информацией с другими участниками платформы, так и от возможности влиять на процессы управления. В современных медиаплатформах обе эти возможности недоступны.",
        "infrastructure": "Мы считаем, что технология блокчейн может предоставить организационную и экономическую инфраструктуру, которая поможет в решении обеих проблем. Публичная и неизменяемая история об экономических операциях и административных действиях, которую каждый может проверить, - отправная точка к тому, чтобы любой пользователь мог понять текущее, а также прошлое состояние платформы. А это позволит каждому составлять суждения, базируясь на неоспоримых фактах и обстоятельствах.",
        "control": "В дальнейшем токены блокчейна позволят объединить использование платформой и управление ею, позволяя каждому принимать участия в решениях на уровне платформы.",
        "signWithCrypto": "Все что нужно для участия в таких мероприятиях - возможность подписывать сообщения цифровым криптографическим ключом. Такое безупречное с точки зрения безопасности взаимодействие предоставляет возможность неограниченному количеству пользователей динамически принимать участие в управлении платформой. Это может служить широкому кругу целей, таких как принятие решений об осуществлении проектов, внутреннему регулированию, участию в формальных ролях или даже в изменении структуры самой организации. Постановка такого взаимодействия на экономические рельсы кажется нам более соответствующей поставленным целям, нежели инфраструктура, построенная на принципах доверительного управления."
      }
    },
    "exit": {
      "title": "Выход",
      "text": {
        "exit": "Выход представляет собой подход полного прекращения взаимодействия с организационной структурой в пользу присоединения к альтернативной, либо создания собственной альтернативы. Эффективность этого подхода зависит от сравнения затрат на прекращение взаимодействия и присоединения к аналогу, либо созданию собственной альтернативы, с желанием терпеть те условия, которые диктует текущая организация. На сегодняшний день стоимость выхода с платформ очень высока.",
        "loweredCosts": "Создание платформ на основе открытого исходного кода, протоколов и публичных данных, как в случае с системами на основе технологии блокчейн, может кардинально снизить эту стоимость. Так как пользователи контролирует собственную информацию, учетную запись и активы, они могут с легкостью присоединиться к новой платформе, даже забрав с собой всех своих друзей.",
        "competition": "Открытые протоколы и наличие открытых данных приведут к развитию возможностей одновременного участия в нескольких платформах одновременно, даже не переключаясь между ними, что будет способствовать конкуренции на уровне платформ [5].",
        "reusability": "Даже более того, использование открытого исходного кода подразумевает, что все наработки платформы могут быть легко использованы повторно, а так как все организационная структура и ее состояние является публичным, можно клонировать целую экосистему и создать альтернативную платформу с альтернативными целями и политиками."
      }
    },
    "ctaText": "В наших планах есть еще много места для участия, поэтому присоединяйтесь, отправив нам <0>email</0> или присоединяйтесь к  <1>Discord</1> или <2>Reddit</2>.",
    "references": {
      "title": "Ссылки",
      "text": "<0>[1] Эрсам, Ф. Токены блокчейна и рассвет децентрализованной бизнес-модели. Medium. 1 августа, 2016.</0><0>[2] Сринивасан Б.С. Мысли о токенах. Medium. 27 мая, 2017.</0><0>[3] Диксон, К. Крипто-токены: прорыв в проектировании открытых сетей. Medium. 1 июня, 2017.</0><0>[4] Хиршман, Альберт О. Выход, голос и лояльность: реакция на упадок фирм, организаций и государств. №25. Издательство Гарвардского университета, 1970.</0><0>[5] Роше, Жан-Шарль и Жан Тироль. «Конкуренция платформ на двусторонних рынках». Журнал европейской экономической ассоциации 1.4 (2003): 990–1029.</0>"
    }
  }
}
