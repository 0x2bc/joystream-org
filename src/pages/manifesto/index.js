import React from 'react';
import fistImage from '../../assets/svg/fist-main.svg';
import Hero from '../../components/Hero';
import LayoutWrapper from '../../components/LayoutWrapper';
import SiteMetadata from '../../components/SiteMetadata';
import BaseLayout from '../../components/_layouts/Base';
import pageData from '../../data/pages/manifesto';
import { pagePropTypes } from '../../propTypes';
import './style.scss';

const ManifestoPage = () => {
  return (
    <BaseLayout className="ManifestoPage">
      <SiteMetadata title="Joystream: A user governed video platform" description="Manifesto" />

      <Hero
        image={fistImage}
        title={pageData.header.title}
        animationStartValue={10}
        animationEndValue={100}
        indent
        theme="blue"
        noOverflow
      >
        <p className="AcropolisPage__hero-paragraph">
          Explore available roles and pick the one that suits you the most. Influence platforms development earning
          Monero in the process.
        </p>
      </Hero>

      <LayoutWrapper>Content</LayoutWrapper>
    </BaseLayout>
  );
};

export { ManifestoPage };
export default ManifestoPage;
