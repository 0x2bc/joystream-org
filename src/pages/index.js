import React from 'react';
import { pagePropTypes } from '../propTypes';

import getApiPath from '../utils/getApiPath';
import mapStatusDataToAnalytics from '../utils/mapStatusDataToAnalytics';
import mapStatusDataToRoles from '../utils/mapStatusDataToRoles';

import withApi from '../components/_enhancers/withApi';

import BaseLayout from '../components/_layouts/Base';
import Link from '../components/Link';
import Subheader from '../components/Subheader';
import TestnetItem from '../components/TestnetItem';
import Analytics from '../components/Analytics';
import Button from '../components/Button';
import TitleWrapper from '../components/TitleWrapper';
import RoleList from '../components/RoleList';
import ColumnsLayout from '../components/ColumnsLayout';
import LayoutWrapper from '../components/LayoutWrapper';
import Hero from '../components/Hero';
import SiteMetadata from '../components/SiteMetadata';

import AcropolisImage from '../assets/svg/acropolis-main.svg';
import heroImage from '../assets/svg/hero-builder.svg';
import RomeImage from '../assets/svg/rome-main.svg';
import constantinopleImage from '../assets/svg/constantinople-main.svg';
import alexandriaImage from '../assets/svg/alexandria-main.svg';
import { ReactComponent as TickImage } from '../assets/svg/tick.svg';
import { ReactComponent as ClockImage } from '../assets/svg/clock.svg';

import { roles } from '../data/pages';
import { launchDate as alexandriaNetworkLaunchDate } from '../data/pages/alexandria';

import './style.scss';

const activeTestnet = {
  name: 'Alexandria',
  incentivesLink: 'https://www.joystream.org/testnet',
};

const IndexPage = ({ content }) => (
  <BaseLayout>
    <SiteMetadata
      title="Joystream: The video platform DAO"
      description="Joystream is a video platform controlled, owned, and operated by its users."
    />
    <Hero
      image={heroImage}
      title={
        <>
          The video <br /> platform DAO
        </>
      }
      animationStartValue={0}
    >
      <p className="IndexPage__hero-paragraph">
        Joystream is a video platform controlled, owned, and operated by its users.
      </p>
      <div className="IndexPage__hero-group">
        <Button noWrap large secondary className="IndexPage__hero-button" href="/testnet/">
          Earn Tokens
        </Button>
        <Button
          noWrap
          large
          secondary
          reversed
          className="IndexPage__hero-button"
          href="https://testnet.joystream.org/"
        >
          Launch UI
        </Button>
      </div>
    </Hero>

    <LayoutWrapper>
      <TestnetItem
        title="Alexandria Testnet"
        image={alexandriaImage}
        children={
          <>Alexandria is our seventh testnet, introducing a variety of important technical improvements.</>
        }
        button={{
          label: 'Explore Alexandria',
          to: '/alexandria',
        }}
      />
    </LayoutWrapper>

    <Analytics title="Testnet Metrics" large content={mapStatusDataToAnalytics(content)}>
      <Button secondary href={activeTestnet.incentivesLink}>
        Participate and Earn Monero
      </Button>
    </Analytics>

    {/*

    <LayoutWrapper>
      <TestnetItem
        title="Alexandria Testnet"
        image={alexandriaImage}
        children="Alexandria is our seventh testnet, focused on a variety of important technical improvements."
        date={alexandriaNetworkLaunchDate}
        button={{
          label: 'Explore Alexandria',
          to: '/alexandria',
        }}
      />
    </LayoutWrapper>

    */}

    <LayoutWrapper dark>
      <TitleWrapper title="Become a user, run the platform">
        <Subheader
          title="Active roles on the current testnet"
          content="learn more, join a role and subscribe for more"
          icon={TickImage}
        />
        <ColumnsLayout>
          <RoleList roles={roles.active} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>
        <Subheader
          title="Roles coming in future testnets"
          content="choose a role, learn more and subscribe to get updated"
          icon={ClockImage}
        />
        <ColumnsLayout columnsCount={3}>
          <RoleList roles={roles.future} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>

        <Button secondary to="/roles" className="IndexPage__roles-button">
          Learn more about the roles
        </Button>
      </TitleWrapper>
    </LayoutWrapper>
  </BaseLayout>
);

IndexPage.propTypes = pagePropTypes;

export { IndexPage };
export default withApi(IndexPage, getApiPath('/'));
