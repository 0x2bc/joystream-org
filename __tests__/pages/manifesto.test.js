import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import { ManifestoPage } from '../../src/pages/manifesto';

describe('ManifestoPage page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ManifestoPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
