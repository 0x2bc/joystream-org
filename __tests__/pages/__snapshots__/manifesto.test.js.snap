// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManifestoPage page renders correctly 1`] = `
<BaseLayout
  className="ManifestoPage"
  lightNavbar={true}
  t={[Function]}
>
  <SiteMetadata
    description="manifesto.siteMetadata.description"
    image="test-file-stub"
    lang="en"
    meta={Array []}
    title="siteMetadata.title"
  />
  <section
    className="ManifestoPage__hero-wrapper"
  >
    <div
      className="ManifestoPage__hero-background"
    >
      <div
        className="ManifestoPage__hero-background__left"
      >
        <icon-component-mock
          className="ManifestoPage__hero-background__left__illustration"
        />
      </div>
      <div
        className="ManifestoPage__hero-background__middle"
      />
      <div
        className="ManifestoPage__hero-background__right"
      >
        <icon-component-mock
          className="ManifestoPage__hero-background__right__illustration"
        />
      </div>
    </div>
    <div
      className="ManifestoPage__hero"
    >
      <header
        className="ManifestoPage__hero__header"
      >
        <h1
          className="ManifestoPage__hero__title"
        >
          manifesto.hero.title
        </h1>
        <p
          className="ManifestoPage__hero__subtitle"
        >
          manifesto.hero.text
        </p>
      </header>
      <icon-component-mock
        className="ManifestoPage__hero__illustration"
      />
    </div>
  </section>
  <section
    className="ManifestoPage__problem-wrapper"
  >
    <div
      className="ManifestoPage__problem ManifestoPage__section"
    >
      <span
        className="ManifestoPage__section-title"
      >
        manifesto.problem.sectionTitle
      </span>
      <h2
        className="ManifestoPage__title ManifestoPage__problem__title"
      >
        manifesto.problem.title
      </h2>
      <div
        className="ManifestoPage__paragraph ManifestoPage__paragraph--top"
      >
        manifesto.problem.text.artAsTool
      </div>
      <div
        className="ManifestoPage__paragraph"
      >
        manifesto.problem.text.mediaAsPrimaryMedium
      </div>
      <div
        className="ManifestoPage__subtitle ManifestoPage__problem__subtitle"
      >
        manifesto.problem.text.however
      </div>
      <div
        className="ManifestoPage__paragraph"
      >
        <span
          className="ManifestoPage__text-highlight ManifestoPage__text-highlight--red"
        >
          manifesto.problem.text.unaccountableInstitutions
        </span>
      </div>
      <div
        className="ManifestoPage__paragraph ManifestoPage__problem__paragraph-4"
      >
        manifesto.problem.text.filteredOnTheirTerms
         
        <span>
          manifesto.problem.text.inflexible
        </span>
      </div>
    </div>
  </section>
  <section
    className="ManifestoPage__goal-wrapper"
  >
    <div
      className="ManifestoPage__section"
    >
      <span
        className="ManifestoPage__section-title ManifestoPage__section-title--white"
      >
        manifesto.goal.sectionTitle
      </span>
      <h2
        className="ManifestoPage__title"
      >
        manifesto.goal.title
      </h2>
      <div
        className="ManifestoPage__paragraph ManifestoPage__paragraph--white ManifestoPage__paragraph--top"
      >
        manifesto.goal.text.arrangement
      </div>
      <div
        className="ManifestoPage__paragraph ManifestoPage__paragraph--white"
      >
        <span
          className="ManifestoPage__text-highlight ManifestoPage__text-highlight--white"
        >
          manifesto.goal.text.accountability
        </span>
      </div>
    </div>
  </section>
  <section
    className="ManifestoPage__thesis-wrapper"
  >
    <div
      className="ManifestoPage__thesis"
    >
      <div
        className="ManifestoPage__section"
      >
        <span
          className="ManifestoPage__section-title ManifestoPage__section-title--blue"
        >
          manifesto.thesis.sectionTitle
        </span>
        <h2
          className="ManifestoPage__title ManifestoPage__title--black"
        >
          manifesto.thesis.title
        </h2>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black ManifestoPage__paragraph--top"
        >
          manifesto.thesis.text.coreThesis
        </div>
      </div>
      <div
        className="ManifestoPage__illustration-wrapper"
      >
        <icon-component-mock
          className="ManifestoPage__illustration"
        />
      </div>
      <div
        className="ManifestoPage__section"
      >
        <h2
          className="ManifestoPage__subtitle ManifestoPage__subtitle--black ManifestoPage__thesis__subtitle"
        >
          manifesto.wedge.title
        </h2>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.wedge.text.alternative
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.wedge.text.blockchainTokens
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.wedge.text.initialHurdle
        </div>
      </div>
      <div
        className="ManifestoPage__illustration-wrapper"
      >
        <icon-component-mock
          className="ManifestoPage__illustration"
        />
      </div>
      <div
        className="ManifestoPage__section"
      >
        <h2
          className="ManifestoPage__subtitle ManifestoPage__subtitle--black ManifestoPage__thesis__subtitle"
        >
          manifesto.accountability.title
        </h2>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.accountability.text.createAndSustain
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.accountability.text.twoWays
        </div>
      </div>
      <div
        className="ManifestoPage__illustration-wrapper"
      >
        <icon-component-mock
          className="ManifestoPage__illustration"
        />
      </div>
      <div
        className="ManifestoPage__section"
      >
        <h2
          className="ManifestoPage__subtitle ManifestoPage__subtitle--black ManifestoPage__thesis__subtitle"
        >
          manifesto.voice.title
        </h2>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.voice.text.improveWithin
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.voice.text.infrastructure
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.voice.text.control
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.voice.text.signWithCrypto
        </div>
      </div>
      <div
        className="ManifestoPage__illustration-wrapper"
      >
        <icon-component-mock
          className="ManifestoPage__illustration"
        />
      </div>
      <div
        className="ManifestoPage__section"
      >
        <h2
          className="ManifestoPage__subtitle ManifestoPage__subtitle--black ManifestoPage__thesis__subtitle"
        >
          manifesto.exit.title
        </h2>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.exit.text.exit
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.exit.text.loweredCosts
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.exit.text.competition
        </div>
        <div
          className="ManifestoPage__paragraph ManifestoPage__paragraph--black"
        >
          manifesto.exit.text.reusability
        </div>
      </div>
    </div>
  </section>
  <section
    className="ManifestoPage__references-wrapper"
  >
    <div
      className="ManifestoPage__references"
    >
      <h2
        className="ManifestoPage__references__title"
      >
        manifesto.references.title
      </h2>
      <div
        className="ManifestoPage__references__items"
      >
        <p
          className="ManifestoPage__references__item"
        >
          manifesto.references.one
        </p>
        <p
          className="ManifestoPage__references__item"
        >
          manifesto.references.two
        </p>
        <p
          className="ManifestoPage__references__item"
        >
          manifesto.references.three
        </p>
        <p
          className="ManifestoPage__references__item"
        >
          manifesto.references.four
        </p>
        <p
          className="ManifestoPage__references__item"
        >
          manifesto.references.five
        </p>
      </div>
    </div>
  </section>
</BaseLayout>
`;
